EventFilter/L1TRawToDigi/python/gtStage2Raw_cfi.py

```diff
@@ -12,7 +12,8 @@
     JetInputTag    = cms.InputTag("simCaloStage2Digis"),
     EtSumInputTag  = cms.InputTag("simCaloStage2Digis"),
     FedId = cms.int32(1404),
-    FWId = cms.uint32(0x10A6),  # FW version in GMT with vtx-etrapolation
+    ## FWId = cms.uint32(0x10A6),  # FW version in GMT with vtx-etrapolation
+    FWId = cms.uint32(0x10F2),  # FW version for packing new HI centrality variables
```

Comment:
@apana @rekovic sorry but is this change backward compatible? This fragment is used by top level configurations for many purposes, not just HI. In all the other updates in this PR I see just addition of pieces, which I would assume is backward compatible.

Reply:
FwId has increased, and the new version only adds records to pack, not remove any existing records, so should be backward compatible.